cmd/uplink/cmd/cp.go

```diff
@@ -100,24 +85,19 @@ func upload(ctx context.Context, bs buckets.Store, srcFile string, destObj *url.
 		bar.Finish()
 	}
 
-	fmt.Printf("Created %s\n", destObj)
+	fmt.Printf("Created %s\n", destObj.String())
```

Comment:
It should default to `String()` method anyways, so the change shouldn't be necessary.